index.html

```diff
@@ -107,7 +107,7 @@
                 </tr>
               </table>
             </td>
-            <td ng-if="sectionName==='taken'">
+            <td ng-if="sectionName==='taken' || tableName==='mana'">
```

Comment:
yeah...but all the if statements are awkward. I guess I just gave up and figured we'd fix them when we eventually migrated to angular 2?